client/src/public/index.html

```diff
@@ -22,22 +22,54 @@
     <meta name="twitter:image" content="/Winden_landscape_logo.png" />
     <meta name="twitter:site" content="@LeastAuthority" />
     <meta name="twitter:card" content="summary_large_image" />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
-      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
-      crossorigin="anonymous"
-      referrerpolicy="no-referrer"
-    />
+    <link rel="stylesheet" href="/nomalize.css" />
     <style>
+      @font-face {
+        font-family: Poppins;
+        src: url(/Poppins/Poppins-Light.ttf) format(truetype);
+        font-weight: 300;
+        font-style: normal;
+      }
+      @font-face {
+        font-family: Poppins;
+        src: url(/Poppins/Poppins-Regular.ttf) format(truetype);
+        font-weight: 400;
+        font-style: normal;
+      }
+      @font-face {
+        font-family: Poppins;
+        src: url(/Poppins/Poppins-Medium.ttf) format(truetype);
+        font-weight: 500;
+        font-style: normal;
+      }
+      @font-face {
+        font-family: Poppins;
+        src: url(/Poppins/Poppins-SemiBold.ttf) format(truetype);
+        font-weight: 600;
+        font-style: normal;
+      }
+      @font-face {
+        font-family: Poppins;
+        src: url(/Poppins/Poppins-Bold.ttf) format(truetype);
+        font-weight: 700;
+        font-style: normal;
+      }
+
+      body {
+        background: linear-gradient(222.19deg, #f0f0f0 23.77%, #ffffff 98.02%);
+        font-size: 16px;
+        color: #282f39;
+        min-width: 320px;
+      }
+
```

Comment:
Do we really need all 5 variants of this font? Do we need a new font here at all?

Reply:
I included the variants to follow the style guide/design docs for winden. AFAIK all weights except for 500 are used right now.